docs/en/Tiered-Locality.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: Tiered Locality
+nickname: Tiered Locality
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+# Tiered Identity
+
+In Alluxio, each entity (masters, workers, clients) has a *Tiered Identity*. A tiered
+identity is an address tuple such as `(node=..., rack=...)`. The elements
+of the tuple are ordered from most specific to least specific, so if two entities have
+the same node name, they are expected to also have the same rack name. Alluxio uses tiered
+identities to optimize for locality. For example, when the client wants to read a
+file from the UFS, it will prefer to read through an Alluxio worker on the same node. 
+If there is no local worker in the first tier (node), the next tier (rack) will be checked 
+to prefer rack-local data transfer. If no workers share a node or rack with the client, an
+arbitrary worker will be chosen.
+
+# Configuration
+
+If the user does nothing to provide tiered identity info, each entity will
+perform a localhost lookup to set its node-level identity info. To explicitly configure
+the value for a locality tier, set the configuration property
+
+```
+alluxio.locality.[tiername]=...
+```
+
+See the [Configuration-Settings](Configuration-Settings.html) page for details on how
+to set configuration properties.
+
+It is also possible to configure tiered identity info via script. To use a script, set
+
+```
+alluxio.locality.script=/path/to/script
+```
+
+The script should be executable and output a comma-separated list of `tierName=...`
+pairs. Here is an example script for reference:
+
+```bash
+#!/bin/bash
+
+echo "host=$(hostname),rack=/rack1"
+```
+
+## Node locality priority order
+
+There are quite a few ways to configure node locality. This is the order of precedence, 
+from highest priority to lowest priority:
+
+1. Set `alluxio.locality.node`
+1. Set `node=...` in the output of the script configured by `alluxio.locality.script`
+1. Set `alluxio.worker.hostname` on a worker, `alluxio.master.hostname` on a master, or
+`alluxio.user.hostname` on a client.
+1. If none of the above are configured, node locality is determined by localhost lookup.
```

Comment:
What's the difference between using `1. 1. 1...` and `1. 2. 3...`?

Reply:
they are equivalent from the perspective of the markdown renderer - I like using `1. 1. 1...` because then you don't need to worry about renumbering